client/src/components/Logo.jsx

```
@@ -1,6 +1,13 @@
+import styles from './Logo.module.css';
+import { useNavigate } from 'react-router-dom';
 
 export default function Logo() {
-  return <img src={logo} alt="" />;
+  const navigate = useNavigate();
+
+  const clickHandler = () => {
+    navigate('/');
+  };
+
+  return <img src="/logo.png" alt="logo" className={styles.logo} onClick={clickHandler} />;
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)